content/en/device-provisioning-service/client-library.md

```diff
@@ -113,38 +140,30 @@ plgd_dps_print_log_fn_t plgd_dps_get_log_fn(void);
 
 A DPS client device is an extension of an [Iotivity-lite](https://github.com/iotivity/iotivity-lite) device. Define your desired Iotivity-lite device and add DPS code to automatically provision the device.
 
-Start up the DPS initialization by calling the `plgd_dps_init(bool skip_ca_verification)` function, which allocates and initializes required data structures.
-Set the `skip_ca_verification` parameter to true to [skip verification of the DPS endpoint](#load-certificates-and-keys), otherwise set it to false.
-
-### Load certificates and keys
-
-To be secure the device communicates with the DPS endpoint using TLS. Use [PKI functions from Iotivity-lite](https://github.com/iotivity/iotivity-lite/blob/master/include/oc_pki.h) to add certificates to device.
-
-Use `oc_pki_add_mfg_cert` function to add manufacturer's certificate and private key to device. This certificate will be used to sign communication with the DPS server, which then can authenticate and identify the device. On success, use the returned credential id to assign a security profile by calling `oc_pki_set_security_profile`. Additionally, assign the credential id value to `dps_mfg_credid` member variable on device context struct.
-
-Use `oc_pki_add_trust_anchor` function to add trusted root certificate of the DPS server, this will be used to verify the DPS server. On success, assign the returned credential id value to `dps_ca_credid` member variable on device context struct.
-
-{{% note %}}
-The verification of the DPS server can be disabled by setting the `skip_ca_verification` argument to false when calling `plgd_dps_init`.
-{{% /note %}}
+Start up the DPS initialization by calling the `plgd_dps_init` function, which allocates and initializes required data structures.
+Use setters `plgd_dps_set_endpoint`, `plgd_dps_set_manager_callbacks`, `plgd_dps_set_skip_verify` and `plgd_dps_set_configuration_resource` to configure the device.
 
 <!--
 ### Set custom logging function
 
 TODO
 -->
 
+### Set DPS Endpoint
+
+To set the DPS endpoint call the `plgd_dps_set_endpoint(plgd_dps_context_t *ctx, const char *endpoint)` function.
```

Comment:
Missing format. Does the setter verify the format? Does it trigger something? How can I retrieve value which is currently configured in the dps? If I call set again, is it updated? What if the device is already provisioned and I call set? Will it trigger reconnection? Or I need to restart? Or reset and start? :)
plantuml would be nice.

Reply:
I know example format is in header file, but please copy paste it here as well. Thanks a lot